### PR TITLE
Docs for installing on GKE

### PR DESCRIPTION
**Pull request by rumpl**
Signed-off-by: Djordje Lukic <djordje.lukic@docker.com>

### CODE REVIEW

File: docs/install-on-gke.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Install on GKE
+
+## Pre-requisites
```
Comment (silvin-lubecki) [inline docs/install-on-gke.md]: Add a note which says you will need a docker CLI master for now, and in the future at least 19.03. We need this PR https://github.com/docker/cli/pull/1583 which has been merged after the 18.09 release.

**Comment by simonferquel** [comment]
relates to #51

File: docs/install-on-gke.md (hunk 1 above)
Comment (simonferquel) [reply docs/install-on-gke.md]: +1

Comment (simonferquel) [reply docs/install-on-gke.md]: We will replace the note with "please use docker CLI 19.03+" when it is out.

Comment (rumpl) [reply docs/install-on-gke.md]: Done

**Comment by silvin-lubecki** [comment]
@simonferquel PTAL

**Pull request merged**